library/IvozProvider/Mapper/Sql/FaxesInOut.php

```diff
@@ -31,9 +31,8 @@ protected function _save(\IvozProvider\Model\Raw\FaxesInOut $model,
         $statusHaschanged = $model->hasChange("status");
 
         if ($isOutgoingFax && $statusHaschanged && $isPending) {
-            $faxSenderJob = new \IvozProvider\Gearmand\Jobs\FaxSender();
-            $faxSenderJob->setFaxInOut($model);
-            $faxSenderJob->send();
+            $ari = new \Asterisk\ARI\Connector();
+            $ari->sendFaxfileRequest($model);
```

Comment:
Given that this is not async anymore, we could throw an error here when something goes wrong